backend/inspirehep/records/indexer/tasks.py

```diff
@@ -17,9 +17,9 @@
 )
 from sqlalchemy.orm.exc import NoResultFound, StaleDataError
 
+from inspirehep.indexer.base import InspireRecordIndexer
+from inspirehep.indexer.utils import get_record
 from inspirehep.records.api import AuthorsRecord, LiteratureRecord
-from inspirehep.records.indexer.base import InspireRecordIndexer
-from inspirehep.records.indexer.utils import get_record
 
```

Comment:
maybe you should remove this since you moved it to indexer/tasks

Reply:
wdym?

Reply:
You added ```indexer/tasks```, why still keep ```records/indexer/tasks```?

Reply:
this is a leftover good catch